src/include/dng/pedigree.h

```diff
@@ -30,12 +30,15 @@
 #include <dng/newick.h>
 #include <dng/read_group.h>
 #include <dng/peeling.h>
+#include <dng/detail/unit_test.h>
 
 namespace dng {
 
 class Pedigree {
 public:
 
+	DNG_UNIT_TEST(test_pedigree_inspect);
+
     enum class TransitionType {
```

Comment:
Probably should move this to the end of the class definition, not the beginning.